app/src/main/java/com/sun_asterisk/moviedb_50/utils/Constant.kt

```diff
@@ -0,0 +1,22 @@
+package com.sun_asterisk.moviedb_50.utils
+
+import com.sun_asterisk.moviedb_50.BuildConfig
+
+object Constant {
+    const val BASE_URL = "https://api.themoviedb.org/3"
+    const val BASE_GENRES_LIST = "/genre/movie/list"
+    const val BASE_MOVIE_BY_ID = "/discover/movie"
+    const val BASE_GENRES_ID = "&with_genres="
+    const val BASE_NOW_PLAYING = "/movie/now_playing"
+    const val BASE_POPULAR = "/movie/popular"
+    const val BASE_TOP_RATE = "/movie/top_rated"
+    const val BASE_UPCOMING = "/movie/upcoming"
+    const val BASE_URL_IMAGE = "https://image.tmdb.org/t/p/w500"
+    const val BASE_LANGUAGE = "&language=en-US"
+    const val BASE_API_KEY = "?api_key=b0d56c5554893fb919ee71c349c8e0f3"
```

Comment:
bạn tìm hiểu thêm về cách giấu api_key nhé, có lẽ mình triển khai chưa đúng hoàn toàn 
https://viblo.asia/p/cac-thu-thuat-de-giau-secret-keypassword-khoi-source-control-va-ung-dung-voi-ci-trong-lap-trinh-android-PwRGgmOnkEd